protected/humhub/modules/notification/components/NotificationManager.php

```diff
@@ -338,6 +338,18 @@ public function setSpaces($spaceGuids, User $user = null)
         ]);
     }
 
+    /**
+     * Reset the notification space settings for all users
+     */
+    public function resetSpaces()
+    {
+        // Reset notifications for all selected membership spaces
+        Membership::updateAll(['send_notifications' => 0]);
+
+        // Delete all selected following spaces
+        Follow::deleteAll(['object_model' => Space::class]);
```

Comment:
Would that delete all existing Space Followings? That would be more than expected. Is that necessary?

Reply:
> Would that delete all existing Space Followings?

Yes, because the table contains the single flag column `send_notifications`:

<img width="851" alt="space_following" src="https://user-images.githubusercontent.com/6995524/107939783-43509a00-6f98-11eb-9995-cc75477ca6a0.png">

But if you think we should not delete the records and can only update the flag to `0`, for that case if in future some new flag/columns will be added there, then we should replace the `deleteAll()` to:

```php
Follow::updateAll(['send_notifications' => 0], ['object_model' => Space::class]);
```

Do you prefer this solution?

Reply:
@yurabakhtin Yes, the solution is better. Then it really only affects the notifications.

Reply:
@luke- Ok, I have fixed that in commit https://github.com/humhub/humhub/pull/4873/commits/7bcbd87149bce7be422f9f4651154a4cea051d39